.buildkite/pipelines/intake.yml

```diff
@@ -15,23 +15,23 @@ steps:
     agents:
       provider: gcp
       image: family/elasticsearch-ubuntu-2004
-      machineType: custom-32-98304
+      machineType: n1-standard-32
```

Comment:
Is there a reason we don't use the newer n2 machines?

Reply:
I definitely want to use the newer families of machines. I was planning to do that when I start looking at resource optimization and such. I have to look at the GCP quotas and such as most of the ones we care about our per-family.